docs/docsite/rst/dev_guide/developing_locally.rst

```diff
@@ -60,35 +60,51 @@ Adding a plugin locally
 =======================
 Ansible loads plugins automatically too, loading each type of plugin separately from a directory named for the type of plugin. Here's the full list of plugin directory names:
 
-    * action_plugins*
-    * cache_plugins
-    * callback_plugins
-    * connection_plugins
-    * filter_plugins*
-    * inventory_plugins
-    * lookup_plugins
-    * shell_plugins
-    * strategy_plugins
-    * test_plugins*
-    * vars_plugins
+    * action*
+    * cache
+    * callback
+    * connection
+    * filter*
+    * inventory
+    * lookup
+    * shell
+    * strategy
+    * test*
+    * vars
```

Comment:
this is incorrect for playbook adjacent and role names, the original was the correct form, you a re add the correct list below, but the one presented here is really never used.

the configuration defaults examples below deal with full paths, that the ending directory does not match is irrelevant

Reply:
> this is incorrect for playbook adjacent and role names, the original was the correct form, you a re add the correct list below, but the one presented here is really never used.

There is a default behavior difference between locally installed (at the machine or user level) and local plugins at the role/playbook level.  That is why I added a new section to distinguish the behavior based on installed defaults. 

Default paths for machine and user level installed plugins are as I have described them. Please reference the values stored in `lib\ansible\config\base.yml`, such as DEFAULT_CONNECTION_PLUGIN_PATH. My edits reflect actual default behavior as installed.

> the configuration defaults examples below deal with full paths, that the ending directory does not match is irrelevant

You are correct, ending paths don't matter for full paths set in variables. My changes did not change those bullet point descriptions.

Reply:
but they are misleading as the directories you now list have no real meaning onto themselves, unlike the previous version in which they are the ones actually inspected automatically for custom plugins

Reply:
I apologize for not understanding the changes made from 2.7 to 2.8. I'll withdraw my request.